qa/integration/src/test/resources/features/user/UserRoleServiceI9n.feature

```diff
@@ -10,8 +10,9 @@
 #     Eurotech - initial API and implementation
 ###############################################################################
 @user
-@integration
 @userRole
+@integration
```

Comment:
Same comment as above; if you wanted to put "integration" tags in the end it's ok, otherwise put it back.